content/en/tracing/troubleshooting/agent_apm_resource_usage.md

```diff
@@ -0,0 +1,44 @@
+---
+title: APM Agent Resource Usage
+kind: Documentation
+aliases:
+  - /tracing/troubleshooting/agent_apm_resource_usage
+---
+
+## How the APM Agent works
+
+The APM Agent is CPU-bound and its CPU usage is correlated with the number of spans received per second.
+
+The APM Agent is designed to buffer unprocessed payloads in memory to avoid data loss. Consequently, throttling the Agent process because of an insufficient CPU limit can lead to out of memory.
+
+## Detect out-of-CPU
+
+The following metrics emitted by APM Agent can help detect out-of-CPU:
+- `datadog.trace_agent.cpu_percent`: CPU usage (in cores), for example, 50 (half a core), 200 (two cores). It can be compared to the [maximum CPU percentage][1] configured for the APM Agent.
```

Comment:
```suggestion
- `datadog.trace_agent.cpu_percent`: CPU usage (in decicores), for example, 50 (half a core), 200 (two cores). It can be compared to the [maximum CPU percentage][1] configured for the APM Agent.
```

Reply:
Nice catch! That was a copy/paste from [here](https://github.com/DataDog/documentation/blob/master/content/en/tracing/troubleshooting/agent_apm_metrics.md), I'll fix both of them.

Reply:
Done!